### PR TITLE
Add Cordova logo

### PR DESCRIPTION
**Pull request by sedroche**
As part of ongoing work to bring mobile services and mobile clients to the OpenShift UI we need to include the cordova icon.

cordova-icon

### CODE REVIEW

**Comment by sedroche** [comment]
@spadgett @rhamilto

**Comment by spadgett** [comment]
/assign @rhamilto 
cc @jennyhaines

@sedroche If you have a color icon, we can also add that to the origin-web-console repo. See https://github.com/openshift/origin-web-console/pull/2208

**Comment by rhamilto** [comment]
This is a nit, but can you move cordova to the end of the list (icomoon puts it at the beginning by default).  This will reduce the diff in selection.json as the iconIndex won't change for every existing icon.

**Comment by sedroche** [comment]
@rhamilto  Sorry missed that in the README. I have added to the end of the list now.

**Comment by rhamilto** [comment]
> @rhamilto Sorry missed that in the README. I have added to the end of the list now.

You didn't miss it.  I added it after reviewing this PR and realizing the instructions should be more clear.  Thanks for your understanding.  :)

File: demo.html [hunk 1]
```diff
@@ -9,10 +9,26 @@
     <link rel="stylesheet" href="style.css"></head>
 <body>
     <div class="bgc1 clearfix">
-        <h1 class="mhmm mvm"><span class="fgc1">Font Name:</span> openshift-logos-icon <small class="fgc1">(Glyphs:&nbsp;70)</small></h1>
+        <h1 class="mhmm mvm"><span class="fgc1">Font Name:</span> openshift-logos-icon <small class="fgc1">(Glyphs:&nbsp;71)</small></h1>
     </div>
     <div class="clearfix mhl ptl">
         <h1 class="mvm mtn fgc1">Grid Size: 16</h1>
+        <div class="glyph fs1">
```
Comment (rhamilto) [inline demo.html]: This leaves Cordova rendering [first in the demo page](https://rawgit.com/sedroche/openshift-logos-icon/edf561cd5e05639e125ff05ceb3ad49da1068d4a/demo.html).  Can you update it to reflect it's new order at the end of the list?

Comment (sedroche) [reply demo.html]: Done.

**Comment by rhamilto** [comment]
Thanks, @sedroche.  Please squash your commits, and I'll merge.

**Comment by sedroche** [comment]
@rhamilto Squashed the extra commit. I was wondering about the version bump and release. How is that handled?

**Comment by rhamilto** [comment]
Thanks, @sedroche!  I'll cut the release now since that requires write access to the repo and publishing access on NPM.

**Comment by sedroche** [comment]
and thank you @rhamilto

**Comment by rhamilto** [comment]
> @sedroche If you have a color icon, we can also add that to the origin-web-console repo. See openshift/origin-web-console#2208

@jennyhaines, I don't think there is an SVG of the Apache Cordova logo available at https://cordova.apache.org/artwork/, but we could use a PNG so Cordova gets a [color equivalent like the others](https://github.com/openshift/origin-web-console/pull/2208).  Thoughts?

**Comment by rhamilto** [comment]
Like this...

screen shot 2018-02-23 at 11 46 31 am
screen shot 2018-02-23 at 11 46 55 am

**Comment by sedroche** [comment]
@rhamilto  Just a fyi that I am aware of this comment. I can create the PR when we get the go ahead. Thanks.

**Comment by jennyhaines** [comment]
@rhamilto @sedroche - I agree with using the PNG for the color Cordova. I looked all over for an SVG, but the only one I found was corrupted, so this is the best we have to go with!

**Comment by jennyhaines** [comment]
I would think that having the referenced file as double the pixel size, and then resizing based on the device viewing it might make sense so it appears clear/not pixelated on retina devices. Does that make sense?

**Comment by spadgett** [comment]
Quick update: @rhamilto has already added the PNG (#2850)

**Comment by spadgett** [comment]
https://github.com/openshift/origin-web-console/pull/2850

**Comment by sedroche** [comment]
:+1: @rhamilto @jennyhaines @spadgett

**Pull request merged**